### ISSUE TITLE
Supoort for i3dm

### ISSUE THREAD

**Issue by YerichoDev** [CesiumGS/cesium-unity#416]
hello, there is any plans to support instanced models in unity ?

**Comment by kring** [comment]
Yes, we plan to support i3dm, which will first require adding it to cesium-native:
https://github.com/CesiumGS/cesium-native/issues/458

We don't have a timeframe for it currently, though.

**Comment by Amit-Yosef** [comment]
Hi There! i've seen that cesium-native now supports i3dm! what would it take to add support in cesium-unity?

**Comment by kring** [comment]
> what would it take to add support in cesium-unity?

I can't give you an exact time frame, but it's coming. You'll likely see i3dm metadata in Unreal first, and then i3dm in Unity after that.

As always, if you're familiar with instanced rendering in Unity and not too scared-away by the C# / C++ conglomeration that is Cesium for Unity, we welcome a pull request that adds it! It's probably not _too_ difficult by using the Unreal implementation as a reference.

**Comment by Patrick-van-Halm-360Fabriek** [comment]
Is there an update on this?
